share/templates/02-Choral/04-SATB_Closed_Score.mscx

```diff
@@ -143,46 +160,36 @@ B</longName>
           <gateTime>100</gateTime>
           </Articulation>
         <Channel name="Tenor">
+          <controller ctrl="0" value="0"/>
+          <controller ctrl="32" value="17"/>
           <program value="52"/>
-          <synti>Fluid</synti>
           </Channel>
         <Channel name="Bass">
+          <controller ctrl="0" value="0"/>
+          <controller ctrl="32" value="17"/>
           <program value="52"/>
-          <synti>Fluid</synti>
           </Channel>
         </Instrument>
       </Part>
     <Staff id="1">
       <VBox>
         <height>10</height>
+        <boxAutoSize>0</boxAutoSize>
         <Text>
           <style>Title</style>
           <text>SATB Closed Score</text>
           </Text>
         </VBox>
       <Measure>
         <voice>
-          <BarLine>
-            <span>1</span>
-            </BarLine>
           <TimeSig>
             <sigN>4</sigN>
             <sigD>4</sigD>
             </TimeSig>
-          <StaffText>
-            <channelSwitch voice="0" name="Soprano"/>
-            <channelSwitch voice="1" name="Alto"/>
-            <channelSwitch voice="2" name="Soprano"/>
-            <channelSwitch voice="3" name="Alto"/>
-            <visible>0</visible>
-            <text>s/a</text>
-            </StaffText>
```

Comment:
why removing this?

Reply:
They should :-(

Reply:
So good I checked ;-)

Reply:
Indeed. The choral templates uses invisible staff texts which I initially missed and added them later. Except for one apperently.

Reply:
These don't really matter, as they are not getting carried over into the score created from the templates though, unfortunately

Reply:
They only do show when opening the template itself directly

Reply:
Agree, it was just the reason I initially missed them. Is already corrected, not pushed yet.

Reply:
Updated.